docs/tutorials/notebooks/basic_ot_between_datasets.ipynb

```diff
@@ -0,0 +1,418 @@
+{
```

Comment:
<p>i am nitpicking, but this is needed, because this is page 1 of the toolbox.</p><p> </p><p>The original introductory nb describes what a point cloud geometry is, and this discussion has been completely removed here. I am not sure we should save space on this, because it's quite important (see e.g. https://github.com/ott-jax/ott/discussions/314)</p><p> </p><p>In particular, since <code>geometry</code> also refers to the module name and base class <code>Geometry</code>,it might be a bit confusing to call your <code>PointCloud</code> object as <code>geometry</code>.Maybe <code>geom</code> is more neutral, or <code>pc_geom</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
What specific information in the pointcloud tutorial is missing here? Unless you want to revert to instantiating a LinearProblem, everything is pretty much there. 

I fixed the name.